chia/wallet/puzzles/did_innerpuz.clvm

```diff
@@ -121,7 +121,7 @@
     (a INNER_PUZZLE my_amount_or_inner_solution)
 
     ; mode 0 - recovery
-    (if (= (sha256tree1 recovery_list_reveal) RECOVERY_DID_LIST_HASH)
+    (if (all (= (sha256tree1 recovery_list_reveal) RECOVERY_DID_LIST_HASH) (> NUM_VERIFICATIONS_REQUIRED 0))
```

Comment:
`(> NUM_VERIFICATIONS_REQUIRED 0)` can probably be just optimised to `NUM_VERIFICATIONS_REQUIRED` - which checks that it is not `0` - though this would potentially allow negative verifications required.

Up to you whether you want to change this.

Reply:
If someone set a negative number mistakenly, I think others shouldn't have permission to spend it.